backend/scripts/load_data.py

```diff
@@ -0,0 +1,62 @@
+import subprocess
+import sqlalchemy as sa
+from os import environ
+from glob import glob
+
+# Optionally, set a data directory to load in 
+data_dir = environ.get("GPKG_DATA_DIR", "sample_data")
+# flag to load data, by default, will load data
+load_data = environ.get("LOAD_GERRY_DB_DATA", "true")
+
+def update_tile_column(engine):
+    """
+    Update the 'tiles_s3_path' column in the 'gerrydbtable' of the public schema.
+
+    This function connects to the database using the provided SQLAlchemy engine
+    and executes an UPDATE query. It sets the 'tiles_s3_path' column to a
+    concatenated string based on the 'name' column.
+
+    Args:
+        engine (sqlalchemy.engine.Engine): SQLAlchemy engine instance for database connection.
+
+    Prints:
+        Success message with the number of updated rows or an error message if the update fails.
+
+    Raises:
+        SQLAlchemyError: If there's an error during the database operation.
+    """
+    print("UPDATING GERRYDB COLUMN")
+    with engine.connect() as connection:
+        try:
+            result = connection.execute(
+                sa.text("UPDATE public.gerrydbtable SET tiles_s3_path = CONCAT('tilesets/', name, '.pmtiles')")
+            )
+            updated_rows = result.rowcount
+            print(f"Successfully updated {updated_rows} rows in gerrydbtable.")
+            connection.commit()
+        except sa.exc.SQLAlchemyError as e:
+            print(f"Error updating gerrydbtable: {str(e)}")
+            connection.rollback()
+
+def load_sample_data():
```

Comment:
H: Looks like you're using two-space indentation from here on? Did you set up the precommits?

Reply:
Good catch, will do

Reply:
Pre-commits are now running on my end. Getting these to play nicely with the monorepo structure and docker is a bit weird, but the docker-compose configuration can manually run the pre-commit